src/Sidekick.Business/Apis/Poe/Parser/ParserService.cs

```diff
@@ -33,16 +33,15 @@ public class ParserService : IOnAfterInit, IParserService
             this.statsDataService = statsDataService;
             this.itemDataService = itemDataService;
             this.patterns = patterns;
+            itemNameTokenizer = new ItemNameTokenizer();
         }
 
-        private Regex NewlinePattern;
-        private Regex SeparatorPattern;
+        private const string NEWLINE_PATTERN = "\r\n";
```

Comment:
There are cases where there is no \r character. Maybe when price checking from the chat, I can't remember the exact scenario. I think the Block separator is fine to not use Regex.

Reply:
Also, the official API item text uses only \n. So if in the future, we want to reparse the text from the API, it would not work.

Reply:
Good points, reverted to regex here.